tests/plugins/test_gen_json.py

```diff
@@ -0,0 +1,150 @@
+
+from unittest import TestCase
+from tempfile import TemporaryDirectory, NamedTemporaryFile
+
+from aac.plugins.gen_json.gen_json_impl import print_json
+from aac.plugins.plugin_execution import PluginExecutionStatusCode, plugin_result
+
+
+class Testgen_json(TestCase):
+
+    def test_print_json_with_output_directory(self):
```

Comment:
This can be addressed with the changes recommended in [this](https://github.com/jondavid-black/AaC/pull/195/files#r798974772) comment (or similar changes).